app/dals/queries_statement/query_params.py

```diff
@@ -0,0 +1,10 @@
+from typing import Dict
+from .TitlesStatement import TitlesStatement
+from .UserJoinedToPostsStatement import UserJoinedToPostsStatement
+from .QueryStatementABC import QueryStatement
+
+# This is where we add another parameters and their corresponding query statements
```

Comment:
Remove commant / make it more professional using docs on funciton (check how to do docstring. Use google format if you choose to do so, I think comments are unnecessary for internal use.